docs/stage-2/README.md

```diff
@@ -1,6 +1,71 @@
-# Bonding Event Architecture
+# Stage 2 APIs
+
 The bonding event is the second stage in the IBCO process and takes place after the user has obtained sEUROs through the bonding curve in Stage 1.
 
+### Operator Stage 2 Public APIs
+
+These are the functions exposed to the user in the Operator Stage 2 contract. This contract can be used to create a bond of sEURO and another token, in order to receive TST once the bond matures.
+
+**showRates()**
+
+*Description*: gets available bond options
+
+*Output*: list of available bond options, as a yield rate and duration (in weeks)
+
+**newBond(address _user, uint256 _amountSeuro, uint256 _rate)**
```

Comment:
maybe you want to amend the api to get this `_user` param removed, as you correctly mentioned before. since we let the user bond fully by themselves now.